### PR TITLE
update minitest style to fix deprecation warnings

### PR DESCRIPTION
**Pull request by evgeni**
draft, as it doesn't fix all warnings yet, but it's a start :)

### CODE REVIEW

**Comment by evgeni** [comment]
@upadhyeammit do you want a redmine for this?

File: test/lib/utils/backup_test.rb [hunk 1]
```diff
@@ -143,15 +143,15 @@ def feature_with_local_method(label, return_value)
...
-        assert !kat_online_backup.fpc_standard_backup?
-        assert !kat_online_backup.fpc_online_backup?
-        assert !kat_online_backup.fpc_logical_backup?
+        refute kat_online_backup.katello_logical_backup?
+        refute kat_online_backup.foreman_standard_backup?
+        refute kat_online_backup.foreman_online_backup?
+        refute kat_online_backup.foreman_logical_backup?
+        refute kat_online_backup.fpc_standard_backup?
+        refute kat_online_backup.fpc_online_backup?
+        refute kat_online_backup.fpc_logical_backup?
```
Comment (evgeni) [inline test/lib/utils/backup_test.rb:154]: this technically was not part of the deprecations, but still good style, imho

**Review by ekohl — COMMENTED** [review]
There are merge conflicts now. I do hope you used [rubocop-minitest](https://docs.rubocop.org/rubocop-minitest/index.html) to generate these changes, though it may be incompatible with the ancient Rubocop version. Perhaps the path should be:
* Declare a minimal Ruby version (both in the gemspec and in Rubocop)
* Update Rubocop
* Add rubocop-minitest

**Comment by ehelms** [comment]
I believe https://github.com/theforeman/foreman_maintain/pull/652 would get us through most of those prerequisites to revive this.

**Comment by ekohl** [comment]
Now that #712 has been merged, there are a ton of merge conflicts.

**Comment by evgeni** [comment]
I'd go as far as saying it fixed all the things this PR also did, so it can be closed!

**Pull request closed without merging**